azure/azure-pr-pipeline.yml

```diff
@@ -38,7 +38,7 @@ extends:
     apigee_deployments:
       - environment: internal-dev
         post_deploy:
-          - template: templates/tests.yml
```

Comment:
Add the tests back in - we want to run the integration tests AND the smoke test, each as a separate step

Reply:
And we want the smoke tests to run on INT and PROD envs (PROD only has smoke tests, not integration tests)

Reply:
@JalfResi I added this to internal-dev for testing. I don't think we can add two templates inside the `post_deploy` step. I made some changes to find a work around by I couldn't.